tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -2611,7 +2611,12 @@ def computeLoadSrd(self, kernel, tP, tc, indices, bpe):
             or isPackedIndex(kernel, idx):
           stride = self.strideRef(tc,idx)
           size =   self.sizeRef(idx)
-          module.add(SSubU32(dst=sgpr(stmp), src0=size, src1=0x1, comment="(size-1)"))
+          if (tP["isA"] and kernel["ProblemType"]["Sparse"] == 1 and idx == 3) or \
```

Comment:
idx == 3 should be if (idx in kernel["ProblemType"]["IndicesSummation"])

Reply:
Thanks for the suggestion, change to use "idx in kernel["ProblemType"]["IndicesSummation]"